types/lru-cache/index.d.ts

```diff
@@ -3,46 +3,47 @@
 // Definitions by: Bart van der Schoor <https://github.com/Bartvds>
 //                 BendingBender <https://github.com/BendingBender>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
 
 declare class LRUCache<K, V> implements Iterable<[K, V]> {
-    constructor(options?: LRUCache.Options<K, V>);
+    constructor(options: LRUCache.Options<K, V>);
```

Comment:
The `Options` parameter is not optional since v7 because `options.max` must be provided. https://github.com/isaacs/node-lru-cache/blob/2be1d2436218360dbf4c673088323b88840bcd24/index.js#L96-L98